app/views/layouts/_user_config.html.erb

```diff
@@ -30,6 +31,17 @@
     if (window.frameElement) { // Hide top bar and footer when loaded within iframe
       document.body.classList.add("hidden-shell");
     }
+    if (userString && userString.length > 0) {
+      const user = JSON.parse(userString)
+      const numTags = JSON.parse(user.followed_tags).length
+      if (numTags < 6) {
+        document.body.classList.add("user-tags-followed-"+numTags);
+      } else {
+        document.body.classList.add("user-tags-followed-max");
+      }
```

Comment:
@benhalpern again, I'm late to the party, but maybe instead of applying specific classes here (especially to `body` element) causing [a bit of mess in main.scss](https://github.com/forem/forem/pull/10868/files#diff-c5a53f74b905bb368fd774f676882b417a22c8d599a5c8097eee51ef8e5d3adfR23-R38), maybe we could just go with inline style here?

```
if (numTags < 6) {
  document.querySelector('#sidebar-nav-followed-tags ').style.height = "205px";
  ...
}
```

Even though it's inline style I think it's gonna be much cleaner this way.